pizzaplace/services/pizza_place_presenter.py

```diff
@@ -0,0 +1,19 @@
+class PizzaPlacePresenter(): 
+  def __init__(self, pizza_place, YelpApi):
+    self.pizza_place = pizza_place
+    self.yelp_api = YelpApi
```

Comment:
:+1: to the instinct to pass in any dependencies this class might need, like whatever class is responsible for talking to the Yelp API. There are other patterns that might help simplify this, like Factories.